lib/foreman_maintain/reporter/cli_reporter.rb

```diff
@@ -62,8 +62,11 @@ def spin
 
         def print_current_line
           @reporter.clear_line
-          line = "#{@spinner_chars[@spinner_index]} #{@current_line}"
-          @reporter.print(line)
+          if @reporter.plaintext?
+            @reporter.print(@current_line.to_s)
+          else
+            @reporter.print("#{@spinner_chars[@spinner_index]} #{@current_line}")
```

Comment:
Should we also call `.to_s` on `@current_line` in else block? I can't find any occurrences where `update` method called with non string param, however its safe to do string typecasting to avoid obvious issues?

Reply:
@upadhyeammit RuboCop complaints: " W: Redundant use of Object#to_s in interpolation."

I think it's fine since the interpolation bit is calling to_s automatically in the background.  I had to change the above line because RuboCop didn't want me using string interpolation on a single variable.